Gdextension (#97)
godot-rust/book: src/gdext/index.md

   
   There is no differentiation between "tool" and "normal" scripts anymore; Rust logic runs as soon as the Godot editor launches.
   godot-rust will likely add an opt-out mechanism to limit execution to the launched game, as that's usually the intended behavior.

>    Currently, if you want to only run when the game runs write this code in you process functions

Reviewer: ```suggestion
   Currently, if you want logic to execute only when the game runs, write this code at the beginning of your process functions:
```

   ```rust
   if Engine::singleton().is_editor_hint() {

>       return

Reviewer: Missing semicolon.
